installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -81,10 +81,21 @@
                 UninstallCommand="/silent /uninstall">
             </ExePackage>
             <MsiPackage
+                Id="InstallPT"
                 DisplayName="Installing PowerToys"
                 SourceFile="$(var.PowerToysPlatform)\Release\$(var.MSIPath)\$(var.MSIName)"
                 Compressed="yes"
-                DisplayInternalUI="no">
+                DisplayInternalUI="no"
+                InstallCondition="NOT Installed">
+                <MsiProperty Name="BOOTSTRAPPERINSTALLFOLDER" Value="[InstallFolder]" />
+            </MsiPackage>
+            <MsiPackage
+                Id="UninstallPT"
+                DisplayName="Uninstalling PowerToys"
+                SourceFile="$(var.PowerToysPlatform)\Release\$(var.MSIPath)\$(var.MSIName)"
+                Compressed="yes"
+                DisplayInternalUI="no"
+                InstallCondition="Installed">
```

Comment:
So will this show on upgrades as well?
If the confusion was with the "Installing PowerToys" string, why not just make it general like "PowerToys" or "Running PowerToys MSI"?

Reply:
Or even just "PowerToys MSI"

Reply:
I like that. That's bulletproof change. Mine is a  it risky

Reply:
fixed

Reply:
Perhaps remove the "Installing" from "Installing Microsoft Edge WebView2". Since we're touching it.

Reply:
makes sense.done